docs/design.html

```diff
@@ -103,7 +103,7 @@ <h3 id="2.2">2.2 Team</h3>
 		<p>
 		Team Representative: Morgenne Besenschek</p>
 		<p>
-		Members: Joshua Varga, Ali Shahid, Dustin Dugal, Michael CWJ, Julius Fan, Kobi Lee, Morgenne Besenschek</p>
+		Members: Joshua Varga, Ali Shahid, Dustin Dugal, Michael CWJ, Julius Fan, Kobi Lee, Morgenne Besenschek, James Robertson</p>
```

Comment:
This is inconsistent with what I have for the back end group, I'm going to email everyone to confirm group members.

Reply:
That would be a correction for another branch, I have always been part of backend